cos_alerter/daemon.py

```diff
@@ -1,33 +1,11 @@
 #!/usr/bin/env python3
 
-import datetime
-import durationpy
-import json
 import signal
 import subprocess
 import sys
-import textwrap
 import time
 
-from .alerter import AlerterState, config, send_notifications
-
-
-def down_interval():
-    return durationpy.from_str(config['watch']['down_interval']).total_seconds()
-
-
-def notify(time_data):
-    if not time.time() - time_data.notify_time > durationpy.from_str(config['notify']['repeat_interval']).total_seconds():
-        #TODO log something here
-        return
-    time_data.notify_time = time.time()
-    last_alert_time = datetime.datetime.fromtimestamp(time_data.alert_time).isoformat()
-    title = '**Alertmanager is Down!**'
-    body = textwrap.dedent(f'''
-        Your Alertmanager instance seems to be down!
-        It has not alerted COS-Alerter since {last_alert_time}.
-        ''')
-    send_notifications(title=title, body=body)
+from .alerter import AlerterState, send_notifications
```

Comment:
I would suggest splitting `send_notifications` out to its own module.

Reply:
I don't really see why. It's such a small function. Any particular reason?